### PR TITLE
Support for gulp package on Windows.

### PR DESCRIPTION
**Pull request by dlebu**

### CODE REVIEW

File: gulpfile.js [hunk 1]
```diff
@@ -4,7 +4,7 @@
 var gulp = require('gulp');
 var log = require('gulp-util').log;
 var sourcemaps = require('gulp-sourcemaps');
-var os = require('os');
+var process = require("process");
```
Comment (MSLaguana) [inline gulpfile.js]: Does the "process" package actually exist? I thought that process was a global

Comment (dlebu) [inline gulpfile.js]: I believe it can be accessed both ways.

Comment (MSLaguana) [inline gulpfile.js]: I tried it out on node 0.12.7 and the "process" package doesn't seem to exist there.

**Comment by MSLaguana** [comment]
Looks good to me

**Comment by joshuaskelton** [comment]
:+1:

File: tools/gulp-extras.js [hunk 2]
```diff
@@ -114,7 +114,7 @@ var executeCommand = function(command, args, callback) {
 
     process.on("error", function(error) {
         if (!errorSignaled) {
-            callback("An error ocurred.");
+            callback("An error ocurred. " + error);
```
Comment (digeff) [inline tools/gulp-extras.js]: The stack trace would be useful too.

Comment (dlebu) [inline tools/gulp-extras.js]: I think we already get that for free.

**Pull request merged**